<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>413</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resource/icon.png</normaloff>:/icons/resource/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>数据库</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>数据库类型：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmb_sql_type">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>主机地址：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_sql_host">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>54</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_sql_username">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>54</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_sql_password">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>54</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>端口：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_sql_port">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>数据库名称：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_sql_database">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_sqlconntest">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>290</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>连接测试</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>服务器</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>固件上传地址：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_firmware_addr">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>271</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>LED模板地址：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_led_template_addr">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>271</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>其他</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>信号查询周期：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ldt_sig_query_period">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_config_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_config_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_config_apply">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
